object-manipulation-1/src/get-keys.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable no-unused-vars */
+function getKeys(object) {
+  const keys = Object.keys(object);
+  return keys;
+}
```

Comment:
Please do not use `Object.keys` for this

Reply:
Sorry for missed instruction on this, I will revising it now.

Reply:
Issue resolved with new commit 
ab67dc5